examples/src/integrationTest/java/org/apache/ignite/example/AbstractExamplesTest.java

```diff
@@ -52,7 +59,16 @@ public void startNode() {
 
         IgnitionManager.init(TEST_NODE_NAME, List.of(TEST_NODE_NAME), "cluster");
 
-        assertThat(ignite, willCompleteSuccessfully());
+        assertThat(igniteFuture, willCompleteSuccessfully());
+
+        ignite = (IgniteImpl) igniteFuture.get(1, TimeUnit.MILLISECONDS);
```

Comment:
You can call "get" without timeout, future is already completed at this point

Reply:
Fix it with join() by Alexander suggestion